AK/Base64.cpp

```diff
@@ -146,7 +146,13 @@ ErrorOr<String> encode_base64(ReadonlyBytes input)
 ErrorOr<ByteBuffer> decode_forgiving_base64(StringView input)
 {
     // 1. Remove all ASCII whitespace from data.
-    auto data = input.trim_whitespace();
+    // FIXME: It is possible to avoid copying input here, it's just a bit tricky to remove the equal signs
```

Comment:
I think these forgiving algorithms in general belong in LibWeb's Infra subfolder, something like LibWeb/Infra/Base64.cpp. For one, the spec's definition of whitespace is a bit different than `is_ascii_space` (See `Infra::is_ascii_whitespace`).

Reply:
Well you're right (turns out the is ascii whitespace definitions are slightly different:D)

But I guess that could be a separate commit and PR

Reply:
Maybe you could add the commit to this PR?

(I'll point out I said the same thing on the PR that added the function in the first place :P)